### PR TITLE
add estimator_status and landed_state fields to AUTOPILOT_STATE_FOR_GD

### PR DESCRIPTION
**Pull request by olliw42**
does https://github.com/mavlink/mavlink/issues/1372

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -6342,6 +6342,8 @@
       <field type="float" name="vz" units="m/s">Z Speed in NED (North, East, Down).</field>
       <field type="uint32_t" name="v_estimated_delay_us" units="us">Estimated delay of the speed data.</field>
       <field type="float" name="feed_forward_angular_velocity_z" units="rad/s">Feed forward Z component of angular velocity, positive is yawing to the right, NaN to be ignored. This is to indicate if the autopilot is actively yawing.</field>
+      <field type="uint16_t" name="estimator_status" enum="ESTIMATOR_STATUS_FLAGS" display="bitmask">Bitmap indicating which EKF outputs are valid.</field>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml:6345]: "EKF outputs" or "Estimator outputs"?

Comment (olliw42) [reply message_definitions/v1.0/common.xml:6345]: I wondered too

Comment (olliw42) [reply message_definitions/v1.0/common.xml:6345]: should I change to Estimator outputs?

Comment (olliw42) [reply message_definitions/v1.0/common.xml]: ok, so changed :)

**Comment by olliw42** [comment]
@julianoes @hamishwillee any reasons which uphold this?

**Comment by julianoes** [comment]
> @julianoes @hamishwillee any reasons which uphold this?

I assume me being on holiday, sorry.

**Comment by olliw42** [comment]
oh, sorry ... I hope you had a great holiday, and that this wasn't a too cruel push back to work

MANY THX!

**Pull request merged**